specific/cmo_project/models/sale.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+
+from openerp import fields, models, api
+
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    project_number = fields.Char(
+        string='Project Code',
+        readonly=True,
+        copy=False,
+    )
+    # version = fields.Integer(
+    #     string='Version',
+    #     size=4,
+    #     default=None,
+    # )
+    project_related_id = fields.Many2one(
```

Comment:
These fields are always editable in Sales Order? I don't see the conditional readonly. (if valid, please fix for all).